app/forms/openid_connect_authorize_form.rb

```diff
@@ -29,8 +29,9 @@ class OpenidConnectAuthorizeForm
   validate :validate_scope
 
   def initialize(params)
-    @acr_values = parse_acr_values(params[:acr_values])
-    simple_attrs = %i(client_id nonce prompt redirect_uri response_type scope state)
+    @acr_values = parse_to_values(params[:acr_values], Saml::Idp::Constants::VALID_AUTHN_CONTEXTS)
+    @scope = parse_to_values(params[:scope], OpenidConnectAttributeScoper::VALID_SCOPES)
+    simple_attrs = %i(client_id nonce prompt redirect_uri response_type state)
```

Comment:
should these values be a constant or something instead? Feels like it, but not sure where they are coming from, exactly.